core/block/editor/state/state.go

```diff
@@ -329,18 +329,18 @@ func (s *State) PickParentOf(id string) (res simple.Block) {
 		if parentId, ok := cache[id]; ok {
 			cacheFound = s.Pick(parentId)
 		}
-		if cacheFound != nil {
-			rootId := s.RootId()
-			topParentId := cacheFound.Model().Id
-			for topParentId != rootId {
-				if nextId, ok := cache[topParentId]; ok {
-					topParentId = nextId
-				} else {
-					cacheFound = nil
-					break
-				}
-			}
-		}
+		// if cacheFound != nil {
```

Comment:
If we just comment this code we will get an avalanche of **discrepancy in state parent search** log messages